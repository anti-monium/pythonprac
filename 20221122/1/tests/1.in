!zxcvbnm,./asdfghjkl;'][poiuytrewqazxswedcvfrtgbnhyujm,kiol./;p[']0