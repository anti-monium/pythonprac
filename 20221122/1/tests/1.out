!']././;p['[p]0asbnbncvcvdfedewfrghhyjkkil;m,m,oiolqaswtgtrujuyzxzx